chain-signatures/contract/src/lib.rs

```diff
@@ -584,6 +584,7 @@ impl VersionedMpcContract {
 #[near_bindgen]
 impl VersionedMpcContract {
     #[handle_result]
+    #[init(ignore_state)]
```

Comment:
Why do we need this change? Not sure if it's safe.

Reply:
ah sorry. shouldn't have pushed this.